probot/test/index.test.js

```diff
@@ -286,7 +292,9 @@ describe('UpdateDB MongoDB methods', async() => {
 
   key = 'db should have removed document if action is closed';
   test(key, async () => {
-    github = await new MockGH(key).gh;
+    github = (recording ? await new GitHubApi(octokitConfig) :
```

Comment:
Could the following work also?
```
    github = await new (recording ? GitHubApi(octokitConfig) : new MockGH(key).gh);
```

Reply:
Not sure yet. I just hit another rate limit. I just pushed a change that I think may help pass the last three tests on the first run. And now I'm realizing I need `await PRtest.deleteMany({})` in `beforeEach`